api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -40,6 +40,21 @@ option java_package = "com.stellarstation.api.v1";
 // station and satellite during the time range.
 service StellarStationService {
 
+  // Cancel a previously reserved upcoming plan. Cancelling the plan will cause it to not be
+  // executed. This action cannot be undone, but it is still possible to reserve the corresponding
+  // pass again by calling `ListUpcomingAvailablePasses` and `ReservePass` as usual - this will
+  // cause a new plan to be reserved. The plan cannot be cancelled less than ten minutes before
+  // AOS.
```

Comment:
What do you think of adding a statement saying "you may be charged even after canceling a pass from API. See our cancellation policy"?

Reply:
Added